src/controllers/graphql/resolvers/reply/replyResolver.js

```
@@ -15,11 +15,11 @@ module.exports = {
     }
   },
   Mutation: {
-    broadcastReply: async (_, { input }, { authenticatedUserInstance } ) => {
+    broadcastReply: async (_, { postPermLink, postAuthor, body }, { authenticatedUserInstance } ) => {
       if (!authenticatedUserInstance) {
         return new AuthenticationError('ERROR_CREATING_OR_EDITING_REPLY');
```

Let's throw this error `throw new AuthenticationError(error message);`